README.md

```diff
@@ -99,6 +99,13 @@ The `"argocd"` field has a few subfields:
   - `"server"` (**Mandatory**): API Server URL for the cluster (Example: `"https://kubernetes.default.svc"`)
   - `"namespace"`: Target namespace in which to deploy the manifests from source (Example: `"my-app-namespace"`)
 
+    > Usually you want to use the same namespace for Applications and AppProjects. For this, use the field `"namespace"`.
+    > 
+    > We also allow to use different namespaces for Applications and AppProjects. In that case **don't** use namespace, use `appProjectNamespace` and `applicationNamespace` instead.
+
+  - `"appProjectNamespace"`: Target namespace in which to deploy the `AppProject` resource.
```

Comment:
The explanation is wrong here, `appProjectNamespace` is the namespaces allowed in the project to deploy to them

Reply:
I think I understand now, I updated it.